etp-backend/src/main/clj/solita/etp/service/valvonta_oikeellisuus/toimenpide.clj

```diff
@@ -45,5 +45,5 @@
 
 (def audit-reply? (partial type? :audit-reply))
 
-(def clears-from-tyojono?
+(def final?
```

Comment:
I kind of feel the old name is more informative, but this is fine as well.

Reply:
ongelma tässä on se että nyt käyttötarkoituksia on kaksi:
- milloin poistetaan valvonta työjonosta
- milloin päivitetään valvonnan oletus valvoja

Reply:
nämä toimenpiteet ovat siis sellaisia jotka päättävät valvonnat eli sen takia muutin tuon final?

Reply:
ajattelin siis että nimi on harhaanjohtava koska tämän perusteella tehdään muutakin kuin vain poistetaan valvonta työjonosta

Reply:
ennen tätä muutosta nimi oli ihan ok koska silloin sen pohjalta ei tehty mitään muuta

Reply:
olisin voinut muuttaa nimeä esim. clears-from-tyojono-and-do-not-update-default-valvoja mutta tämä ei oikein skaalaudu jos nimeen yritetään laittaa logiikka joka seuraa siitä että valvonta päättyy. Parempi että on lyhyt nimi joka kuvaa sen että tämä on valvonnan lopettava toimenpide ja taas jos haluaa etsiä mitä siitä seuraa niin etsii kohdat joissa tätä predikaattia käytetään.

Reply:
sinänsä en tiedä onko final? hyvä vai pitäisikö olla esim. close-valvonta? niin olisi yhdenmukainen hakujen include-closed lipun kanssa.